user/cmd/user/main.go

```
@@ -55,9 +46,9 @@ func main() {
 		logger.Warn("error finding config file, using default values", zap.Error(err))
 	}
 
-	gwPort := viper.Get("gatewayPort").(string)
+	gwPort := viper.GetString("gateway.port")
 	ctx := context.Background()
-	key := option.WithCredentialsFile(viper.Get("secretPath").(string))
+	key := option.WithCredentialsFile(viper.GetString("secret.path"))
 
 	app, err := firebase.NewApp(ctx, nil, key)
 	if err != nil {
```

use panic

---

panic instead of logger warn on line 46? or to use panic instead of logger in main.go in general?